### PR TITLE
new icon: cmake (original, plain, original-wordmark, plain-wordmark)

### LINKED ISSUE #880: Icon request: CMake
### About the icon

CMake generates native build tool files specific to a compiler and platform

### Links and sources

CMake website: https://cmake.org
CMake Logo (png): https://cmake.org/wp-content/uploads/2018/11/cmake_logo_slider.png

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by maltejur**
image

Closes #880 

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date

You can compare the logo with https://cmake.org:
image

### CODE REVIEW

**Comment by maltejur** [comment]
@Thomas-Boi didn't the bot have the feature to automatically optimize the SVGs at some point? What happened to that?

**Comment by Thomas-Boi** [comment]
After checking the python file, it turns out that I still have a check for the `x` and `y` attributes. This is because the old SVGO (the minifier tool) didn't have a way for me to remove these attributes. I might have also missed it as well but I doubt it. However, seems like SVGO now has an option to remove these attributes by patterns. Thus, I can now remove the `x` and `y` checks.

Summary: this bug will be fixed sometime in the future. For now, you just have to make sure that the `<svg>` doesn't have any `x` or `y` attributes.

**Comment by maltejur** [comment]
Ah, good to know.

**Comment by maltejur** [comment]
> This path doesn't exist: D:\a\devicon\devicon\icons\cmake\cmake-plain.svg

Hmm weird that the check workflow didn't catch that.

**Comment by Thomas-Boi** [comment]
> > This path doesn't exist: D:\a\devicon\devicon\icons\cmake\cmake-plain.svg
> 
> Hmm weird that the check workflow didn't catch that.

You are right 🤔. I think I didn't do name checking, just checking the content of the files. I'll make sure to get this feature in.

**Comment by Thomas-Boi** [comment]
@maltejur can you rename all 4 files so it matches one of the below format please?
image

So `cmake-icon-original` should just be `cmake-original` etc.

**Comment by Thomas-Boi** [comment]
Sweet, your icon looks great @maltejur 👍 . Thank you for your work and I'll make sure to implement the features to `check-svg` by the end of the year.

**Review by Thomas-Boi — APPROVED** [review]

File: icons/cmake/cmake-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...3.6.1 1.3.3 1.9.6l-1.1 3.5c-.5-.4-1.2-.6-1.8-.8-.7-.2-1.5-.3-2.2-.3-2 0-3.9.7-5.3 2-1.7 1.4-2.7 3.2-3.1 5.2zM75.4 56.4h6.4l-4.2 19.9h-3.8l3.5-16.5-8.9 16.5h-3.8l-1.9-16.6-3.5 16.6h-3.8l4.2-19.9H66l1.5 15.1 7.9-15.1zM91.1 74c-.7.9-1.6 1.7-2.6 2.2-.8.4-1.7.6-2.6.6-1.3.1-2.5-.3-3.4-1.2-.7-.8-1-1.9-.8-3 .3-1.6 1.2-2.9 2.6-3.8 1.7-1 3.7-1.4 5.7-1.3h2.4l.1-.3c.2-1 .1-1.8-.4-2.2-.7-.5-1.5-.7-2.3-.7-.8 0-1.7.2-2.5.5-.8.2-1.6.6-2.3 1.2l.6-3.3c.8-.3 1.7-.6 2.6-.8.9-.2 1.9-.4 2.8-.4 2.2 0 3.7.5 4.5 1.5.8 1.1 1 2.7.5 4.9l-.9 3.9-.6 3.3c-.1 1-.2 1.4-.2 1.4h-3.4v-.7c-.1-.6 0-1.2.2-1.8zm.7-4.1.1-.3h-1.7c-1.1 0-2.2.2-3.2.7-.7.3-1.2 1-1.3 1.7-.1.5 0 1 .3 1.4.5.4 1.1.5 1.7.5 1.1 0 2.1-.4 2.9-1.2.5-.9.9-1.8 1.2-2.8zM102.2 55h3.8l-2.7 12.7 5.9-6h4.6l-6.8 6.6 4.5 8.1h-4.9l-3.4-7.2-1.5 7.2h-3.8l4.3-21.4zM125.5 75.6c-.8.4-1.6.7-2.4.8-.9.2-1.8.3-2.8.3-2.6 0-4.4-.7-5.4-2s-1.3-3.2-.8-5.7c.4-2.1 1.4-4.1 3-5.5 1.5-1.4 3.4-2.1 5.4-2.1 2.5 0 4.1.8 4.8 2.3.8 1.5.9 3.7.2 6.6h-10.1c-.1.8.1 1.7.5 2.4.7 1 1.8 1.5 3 1.3 1 0 2-.2 3-.6.7-.2 1.4-.6 2.1-1l-.5 3.2zm-7.6-8h6.6c.3-.9.1-1.8-.4-2.6-.5-.6-1.3-.9-2.1-.9-1.1 0-2.1.4-2.8 1.2-.5.8-.9 1.5-1.3 2.3z"/><g><path fill="#CDCDCE" d="m11.3 71.6 8.1 3.2-.8-9.4z"/></g></svg>
```
Comment (Panquesito7) [inline icons/cmake/cmake-original-wordmark.svg:1]: These files are missing an endline 😄 I'll make a PR to fix that.

**Pull request merged**